### PR TITLE
Support for some currencies

### LINKED ISSUE #1104: "\textunderscore" throws "Expected 'EOF'"-error
Hi,

when I'm writing `\textunderscore` into the [demo editor](https://khan.github.io/KaTeX/), it gives me the following error message: `KaTeX parse error: Expected 'EOF', got '\textunderscore' at position 1: \̲t̲e̲x̲t̲u̲n̲d̲e̲r̲s̲c̲o̲r̲e̲`.

It seems as if it isn't supported anymore, but it's [still in the documentation](https://khan.github.io/KaTeX/function-support.html#symbols) (first column, 6th row).

### PR DESCRIPTION
**Pull request by Syndesi**
Resolves #1104 , at this moment not tested.

### CODE REVIEW

**Comment by Syndesi** [comment]
[clabot:check]

File: src/symbols.js [hunk 1]
```diff
@@ -437,6 +437,18 @@ defineSymbol(math, main, textord, "\u2018", "`");
...
+defineSymbol(text, main, textord, "€", "\\texteuro");
+defineSymbol(math, main, textord, "¥", "\\¥");
+defineSymbol(text, main, textord, "¥", "\\¥");
+defineSymbol(text, main, textord, "¥", "\\textyen");
+defineSymbol(math, main, textord, "₹", "\\₹");
+defineSymbol(text, main, textord, "₹", "\\₹");
+defineSymbol(text, main, textord, "₹", "\\textrupee");
+defineSymbol(math, main, textord, "₽", "\\₽");
+defineSymbol(text, main, textord, "₽", "\\₽");
+defineSymbol(text, main, textord, "₽", "\\textruble");
```
Comment (kevinbarabash) [inline src/symbols.js]: I thought you'd be adding:
```
\textbar
\textbardbl
\textbraceleft
\textbraceright
\textdagger
\textdaggerdbl
\textdollar
\textellipsis
\textemdash
\textendash
\textgreater
\textless
\textquotedblleft
\textquotedblright
\textquoteleft
\textquoteright
\textsterling
\textunderscore
```
which we already have symbols for.  Unfortunately, we don't have any fonts with with glyphs for the Euro, Rupee, or Ruble.  The Yen is already supported.  Also, defining symbols with unicode support should look like:
```
defineSymbol(math, ams, textord, "\u00a5", "\\yen", true);
```

**Comment by kevinbarabash** [comment]
Blocked on https://github.com/KaTeX/katex-fonts/issues/18.

**Comment by kevinbarabash** [comment]
Please don't delete package-lock.json.  What version of `npm` are you using?

**Comment by kevinbarabash** [comment]
@Syndesi I'm going to close this for now.  Please re-open https://github.com/KaTeX/katex-fonts/issues/18 is resolved.  I'm going to try to add some documentation to https://github.com/KaTeX/katex-fonts to help people add new symbols to the fonts.  I'll let you know when that documentation is ready if you're interested in giving that a try.

**Pull request closed without merging**